Dockerfile

```diff
@@ -19,156 +20,330 @@
-    GO_VERSION=1.16.5 \
-    HASKELL_STACK_VERSION=2.1.3 \
-    NPM_VERSION=7.20.6 \
-    PYTHON_PIPENV_VERSION=2018.11.26 \
-    PYTHON_VIRTUALENV_VERSION=15.1.0 \
-    SBT_VERSION=1.6.1 \
-    YARN_VERSION=1.22.10 \
-    # SDK versions.
-    ANDROID_SDK_VERSION=6858069 \
-    # Installation directories.
-    ANDROID_HOME=/opt/android-sdk \
-    GOPATH=$HOME/go
-
-ENV DEBIAN_FRONTEND=noninteractive \
-    PATH="$PATH:$HOME/.local/bin:$GOPATH/bin:/opt/go/bin:$GEM_PATH/bin:/opt/ort/bin"
+    . /opt/nodejs/nvm.sh \
+    && GRADLE_USER_HOME=/tmp/.gradle/ \
+    && scripts/import_proxy_certs.sh \
+    && scripts/set_gradle_proxy.sh \
+    && sed -i -r 's,(^distributionUrl=)(.+)-all\.zip$,\1\2-bin.zip,' gradle/wrapper/gradle-wrapper.properties \
+    && sed -i -r '/distributionSha256Sum=[0-9a-f]{64}/d' gradle/wrapper/gradle-wrapper.properties \
+    && ./gradlew --no-daemon --stacktrace -Pversion=$ORT_VERSION :cli:distTar :helper-cli:startScripts
+
+RUN mkdir -p /opt/ort \
+    && tar xf /usr/local/src/ort/cli/build/distributions/ort-$ORT_VERSION.tar -C /opt/ort --strip-components 1 \
+    && cp -a /usr/local/src/ort/scripts/*.sh /opt/ort/bin/ \
+    && cp -a /usr/local/src/ort/helper-cli/build/scripts/orth /opt/ort/bin/ \
+    && cp -a /usr/local/src/ort/helper-cli/build/libs/helper-cli-*.jar /opt/ort/lib/ \
+    && cd \
+    && rm -rf /usr/local/src
+
+#------------------------------------------------------------------------
+# Main container
+FROM eclipse-temurin:11-jdk-focal
+
+ENV LANG=en_US.UTF-8
+ENV LANGUAGE=en_US:en
+ENV LC_ALL=en_US.UTF-8
+ENV HOME=/workspace
+
+# Run with non privileged user
+RUN groupadd --gid 1000 ort \
+    && useradd --uid 1000 --gid ort --shell /bin/bash --home-dir /workspace --create-home ort
+
+COPY docker/00-add_local_path.sh /etc/profile.d/
+
+# Python
+COPY --chown=ort:ort --from=pythonbuild /opt/python /opt/python
+COPY --from=pythonbuild /etc/profile.d/python.sh /etc/profile.d/
+
+# Ruby
+COPY --chown=ort:ort --from=rubybuild /opt/rbenv /opt/rbenv
+COPY --from=rubybuild /etc/profile.d/ruby.sh /etc/profile.d/
+
+# NodeJS
+ARG NODEJS_VERSION=16.13.2
+ARG NVM_DIR=/opt/nodejs
+ENV NODE_PATH $NVM_DIR/v$NODEJS_VERSION/lib/node_modules
+ENV PATH $NVM_DIR/versions/node/v$NODEJS_VERSION/bin:$PATH
+COPY --chown=ort:ort --from=nodebuild /opt/nodejs /opt/nodejs
+
+# Golang
+COPY --chown=ort:ort --from=gobuild /opt/go /opt/go/
+COPY --from=gobuild  /etc/profile.d/go.sh /etc/profile.d/
+
+# Haskell
+COPY --from=haskellbuild /usr/bin/stack /usr/bin
+
+# Repo and Android
+ENV ANDROID_HOME=/opt/android-sdk
+ENV ANDROID_API_LEVEL=29
+COPY --from=androidbuild /usr/bin/repo /usr/bin/
+COPY --from=androidbuild /etc/profile.d/android.sh /etc/profile.d/
+COPY --chown=ort:ort --from=androidbuild /opt/android-sdk /opt/android-sdk
+
+# ScanCode
+COPY --chown=ort:ort --from=scancodebuild /opt/scancode /opt/scancode
+RUN ln -s /opt/scancode/bin/scancode /usr/bin/scancode \
+    && ln -s /opt/scancode/bin/pip /usr/bin/scancode-pip \
```

Comment:
I don't think need need this in `PATH`.

Reply:
Removed. Now installed by pip on last commit